### PR TITLE
Renamed typography to fonts

### PR DESCRIPTION
**Pull request by girish-lokapure**
PRESS2-1070 : Replace technically accurate "typography" with more commonly understood "fonts"

### CODE REVIEW

File: src/OnboardingSPA/components/Drawer/DrawerPanel/index.js [hunk 1]
```diff
@@ -35,6 +35,7 @@ import { store as nfdOnboardingStore } from '../../../store';
 import WithDesignBack from './WithDesignBack';
 
 const DrawerPanel = () => {
+	// eslint-disable-next-line no-unused-vars
```
Comment (officiallygod) [inline src/OnboardingSPA/components/Drawer/DrawerPanel/index.js:38]: Why are we still using this state? Please check and remove it if possible, I don't see any issue.

File: src/OnboardingSPA/components/Drawer/DrawerPanel/index.js [hunk 2]
```diff
@@ -45,6 +46,7 @@ const DrawerPanel = () => {
 		};
 	}, [] );
 
+	// eslint-disable-next-line no-unused-vars
```
Comment (officiallygod) [inline src/OnboardingSPA/components/Drawer/DrawerPanel/index.js:49]: Same, Let's not use things we don't need, Thanks.

File: src/OnboardingSPA/components/Drawer/DrawerPanel/index.js [hunk 3]
```diff
@@ -64,6 +66,7 @@ const DrawerPanel = () => {
 	}, [ drawerView ] );
 
 	return (
+		// eslint-disable-next-line jsx-a11y/no-static-element-interactions
```
Comment (officiallygod) [inline src/OnboardingSPA/components/Drawer/DrawerPanel/index.js:69]: Let us do this like we have done [here](https://github.com/newfold-labs/wp-module-onboarding/blob/4406cb9de1ef268935d340ed60217c3e8350acd4/src/OnboardingSPA/components/Drawer/DrawerPanel/DesignColors.js#L238C6-L241C51).

**Review by officiallygod — CHANGES_REQUESTED** [review]

File: includes/Data/Events.php [hunk 4]
```diff
@@ -29,7 +29,7 @@ final class Events {
 		'primary_type_set'            => true,
 		'secondary_type_set'          => true,
 		'theme_style_selected'        => true,
-		'typography_selected'         => true,
+		'fonts_selected'              => true,
```
[the remark this answers is not in the document]
Comment (girish-lokapure) [reply includes/Data/Events.php:32]: Hiive needs to be updated about the same

**Review by officiallygod — APPROVED** [review]
LGTM

**Review by arunshenoy99 — APPROVED** [review]
Should we get the `languages/wp-module-onboarding.pot` file? Seeing merge conflicts as well @girish-lokapure. Rest looks good, let's take care of these before merging

**Comment by officiallygod** [comment]
> Should we get the `languages/wp-module-onboarding.pot` file updated? Seeing merge conflicts as well @girish-lokapure. Rest looks good, let's take care of these before merging

Looks like we need the pot file to be updated cause the text strings do change.

**Comment by girish-lokapure** [comment]
> Should we get the `languages/wp-module-onboarding.pot` file updated? Seeing merge conflicts as well @girish-lokapure. Rest looks good, let's take care of these before merging

this is done

**Review by arunshenoy99 — COMMENTED** [review]
Looks good, @girish-lokapure. Let's hold off on merging this until we have a stable HostGator release.

**Comment by girish-lokapure** [comment]
@arunshenoy99 please review the dependency module as well:
https://github.com/newfold-labs/wp-module-onboarding-data/pull/4

**Review by arunshenoy99 — APPROVED** [review]

**Comment by arunshenoy99** [comment]
We need to fix the lint on this.

**Pull request merged**